### PR TITLE
Proof read ALARM_AND_ABORT

### LINKED ISSUE #191: Proof ALARM_AND_ABORT
Proof read transcribed code of [ALARM_AND_ABORT](//github.com/chrislgarry/Apollo-11/blob/master/Comanche055/ALARM_AND_ABORT.agc) against scans

Lines: 231
Page: 1493—1496

Reduced quality scans can be found [here](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055)

### PR DESCRIPTION
**Pull request by axelstudios**
This should close #191.

There was only one real mistake, but I also adjusted blank lines to more closely match the source scans.

Also, the comma in `EXAMPLE FOLLOWS,` on line R0051 on page [1494](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1494.jpg) looks very close to a period, but based on details from previous pull requests I think it probably is just a rough comma.

### CODE REVIEW

File: Comanche055/ALARM_AND_ABORT.agc [hunk 1]
```diff
@@ -36,10 +36,10 @@
 # EITHER IN INTERRUPT OR UNDER EXECUTIVE CONTROL.
 #
 # 	THE CALLING SEQUENCE IS AS FOLLOWS:
```
Comment (wopian) [inline Comanche055/ALARM_AND_ABORT.agc]: ✨ Remove `THE` here

File: Comanche055/ALARM_AND_ABORT.agc [hunk 2]
```diff
@@ -36,10 +36,10 @@
...
+#
 #		TC	ALARM
 #		OCT	NNNNN
 #					# (RETURNS HERE)
-
```
Comment (wopian) [inline Comanche055/ALARM_AND_ABORT.agc]: ✔️

File: Comanche055/ALARM_AND_ABORT.agc [hunk 3]
```diff
@@ -109,6 +109,7 @@ MULTFAIL	CA	L
 #		CAF	OCTXX		# ALARM CODE
 #		TC	BANKCALL
 #		CADR	PRIOLARM
+#
```
Comment (wopian) [inline Comanche055/ALARM_AND_ABORT.agc]: ✔️

File: Comanche055/ALARM_AND_ABORT.agc [hunk 4]
```diff
@@ -133,7 +134,6 @@ PRIOLARM	INHINT			# * * * KEEP IN DISPLAY ROUTINES BANK
 		TCF	PRIODSPR
 
 # Page 1495
-
```
Comment (wopian) [inline Comanche055/ALARM_AND_ABORT.agc]: ✔️

File: Comanche055/ALARM_AND_ABORT.agc [hunk 5]
```diff
@@ -163,7 +163,7 @@ WHIMPER		CA	TWO
 		SETLOC	FFTAG7
 		BANK
 
-P00DOO		INHINT
+POODOO		INHINT
```
Comment (wopian) [inline Comanche055/ALARM_AND_ABORT.agc]: ✔️Will need a mass rename of `P00DOO` -> `POODOO` at some point

File: Comanche055/ALARM_AND_ABORT.agc [hunk 6]
```diff
@@ -184,7 +184,6 @@ OCT77770	OCT	77770		# DONT MOVE
 		ADRES	STATEFLG
 
 		TC	DOWNFLAG
-
```
Comment (wopian) [inline Comanche055/ALARM_AND_ABORT.agc]: ✔️

File: Comanche055/ALARM_AND_ABORT.agc [hunk 7]
```diff
@@ -211,6 +210,7 @@ OCT217		OCT	00217
 
 DOALARM		EQUALS	ENDOFJOB
 # CALLING SEQUENCE FOR VARALARM
+#
```
Comment (wopian) [inline Comanche055/ALARM_AND_ABORT.agc]: ✔️

**Comment by axelstudios** [comment]
Good catch, I'm surprised I missed that extra 'The'..  I amended the original commit.

**Comment by wopian** [comment]
Thanks ❤️ Always nice to see new contributors 👍

**Comment by chrislgarry** [comment]
@axelstudios @wopian thanks guys!

**Pull request merged**